packages/core/bridge/providers/cctp/cctpBridge.ts

```diff
@@ -29,6 +31,8 @@ export async function cctpBridgeTokenFromSolana(params: IBridgeFromSolanaParams)
         solanaRpcUrl
     } = params;
 
+    const lockerSolanaSigner = await createLockerSolanaSigner(solanaSigner);
```

Comment:
A LockerSolanaSigner should be supplied to cctpBridgeTokenFromSolana. cctpBridgeTokenFromSolana shuoult not have to convert solanaSinger -> lockerSolanaSigner.